app/bundles/DashboardBundle/Tests/Controller/DashboardControllerFunctionalTest.php

```diff
@@ -112,10 +80,14 @@ public function testWidgetWithSegmentBuildTime(): void
         $this->em->flush();
         $this->em->detach($widget);
 
-        $this->client->xmlHttpRequest('GET', sprintf('/s/dashboard/widget/%s', $widget->getId()));
-        $this->assertResponseIsSuccessful();
+        $this->client->request('GET', sprintf('/s/dashboard/widget/%s', $widget->getId()), [], [], [
+            'HTTP_X-Requested-With' => 'XMLHttpRequest',
+        ]);
+
+        $response = $this->client->getResponse();
+        Assert::assertSame(200, $response->getStatusCode());
```

Comment:
Why was this reverted? `$this->assertResponseIsSuccessful();` also shows the request info when the assert fails. With this you get no information to debug.

Reply:
I explained the reason in the other note. I believe it is resolved.